appdatabase/migrations/sql/0020_metrics.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS app_metrics (
+       id INTEGER PRIMARY KEY,
```

Comment:
I would probably leave a comment here as we are aliasing `rowid` and it might not be immediately clear if you don't know the behavior of sqlite (or we can just drop it and use rowid, but that's up to you).

Reply:
We can drop it. The id is not important.